src/axolotl/prompt_strategies/vicuna.py

```diff
@@ -0,0 +1,176 @@
+"""
+Prompt Strategy for finetuning Vicuna v1.1+ models
+see also https://huggingface.co/psmathur/orca_mini_v2_7b for more information
+
+Use dataset type: "vicuna" in conig.yml to use this prompt style.
+
+E.g. in the config.yml:
+```
+datasets:
+  - path: vicuna_finetune_train.jsonl
+    type: vicuna
+```
+
+The dataset itself should look like this:
+```
+{'conversation':[{"from": "human", "value": "Who are you?"}, {"from": "gpt", "value": "I am Vicuna"},...]}
```

Comment:
I'm not so clear on this. Should it be `conversations` with `s`? This will allow it to be compatible with older sharegpt datasets.